docs/DeveloperGuide.md

```diff
@@ -1032,17 +1038,39 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
+### Adding an applicant 
+
+1. Adding an applicant while all applicants are being shown 
+
+    1. Prerequisites: List all persons using the `list` command. Multiple persons in the list. 
```

Comment:
There are still instances of `persons` here 😅. I think should be changed to `applicants`?

Reply:
Yep my bad, I've changed them!